packages/relay/src/formatters.ts

```diff
@@ -288,6 +288,16 @@ const isHex = (value: string): boolean => {
   return hexRegex.test(value);
 };
 
+// Returns the sum of all transfer amounts for the specified account. The amount is negative if the account is charged,
+// it is positive if the account is receiving it, thus the amount is first negated and then added to the sum.
+const getTransferAmountSumForAccount = (transactionRecord, accountId: undefined | string): number => {
```

Comment:
It makes no sense to accept `undefined` values for `accountId` here:
```suggestion
const getTransferAmountSumForAccount = (transactionRecord, accountId: string): number => {
```